benchexec/tools/two_ls.py

```diff
@@ -42,8 +50,8 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         elif returncode == 0:
             status = result.RESULT_TRUE_PROP
         elif returncode == 10:
-            if len(output) > 0:
-                result_str = output[-1].strip()
+            if len(run.output) > 0:
```

Comment:
Should be written as `if run.output:`

Reply:
fixed in 2b4a6a1fbf69d98062f433ab690e55db316abb02